package.json

```diff
@@ -122,9 +123,9 @@
     "rimraf": "^3.0.2",
     "shortid": "^2.2.16",
     "solhint": "^3.3.7",
-    "solidity-coverage": "^0.8.0-beta.1",
+    "solidity-coverage": "0.8.0-beta.1",
     "solidity-steamroller": "^1.1.0",
-    "truffle": "^5.5.30",
+    "truffle": "5.5.30",
```

Comment:
Why the pins?

Reply:
Because unfortunately it's only a very special version of truffle / solidity-coverage that work.